### DISCUSSION TITLE
Ability to scale recipe based on number of servings

### DISCUSSION THREAD

**Discussion by enrauk** [mealie-recipes/mealie#3880] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Allow users to change the number of potions required to recalculate the ingredient quantities needed.  Default the number of portions required to the recipe number of servings.

### Please provide a concise description of the feature that would resolve your issue.

Would allow users to see the quantity of ingredients required based on the number of servings needing to be prepared

### Please consider and list out some caveats or tradeoffs made in your design decision

This is fairly standard functionality in most recipe managers

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
This has been a feature for a long time. Are your recipes parsed?

https://docs.mealie.io/documentation/getting-started/faq/#how-do-i-enable-smart-ingredient-handling
